common/src/main/java/dev/architectury/hooks/item/tool/AxeItemHooks.java

```diff
@@ -0,0 +1,19 @@
+package dev.architectury.hooks.item.tool;
+
+import com.google.common.collect.ImmutableMap;
+import net.minecraft.world.item.AxeItem;
+import net.minecraft.world.level.block.Block;
+
+import java.util.HashMap;
+
+public final class AxeItemHooks {
+    private AxeItemHooks() {
+    }
+
+    public static void addStrippingRecipe(Block src, Block dst) {
+        if (AxeItem.STRIPPABLES instanceof ImmutableMap) {
+            AxeItem.STRIPPABLES = new HashMap<>(AxeItem.STRIPPABLES);
+        }
+        AxeItem.STRIPPABLES.put(src, dst);
```

Comment:
Does the axe item code still assume the block has `AXIS` (ie is a `PillarBlock`)?

Reply:
Yes, it assumes both the source block _and_ the destination block have the `AXIS` property.

Reply:
It'd probably be good to add checks here already so that it crashes already at registration time instead of when the recipe is used.